.github/workflows/uvloop_test.yml

```diff
@@ -0,0 +1,45 @@
+name: test uvloop
+
+on:
+  push:
+    paths-ignore:
+      - "docs/**"
+      - "*.md"
+
+  pull_request:
+    paths-ignore:
+      - "docs/**"
+      - "*.md"
+
+jobs:
+  build:
+    # We want to run on external PRs, but not on our own internal PRs as they'll be run
```

Comment:
what does internal vs external mean in this context?

Reply:
This is copy pasta to maintain other CI behavior - this stops running the same CI twice for maintainers like me who make a branch and push to the black repo itself. @ichard26 did this and I like that it saves some resources. GitHub should really default to this to save CPU. One of us should go work there and claim the reward points.